Lib/test/test_inspect/test_inspect.py

```diff
@@ -1892,8 +1921,13 @@ def function():
             inspect.ismethoddescriptor(Owner.__dict__['a_property']),
             'a property is not a method descriptor')
         self.assertFalse(
-            inspect.ismethoddescriptor(Slotermeyer.__dict__['a_slot']),
-            'a slot is not a method descriptor')
+            inspect.ismethoddescriptor(member_descriptor),
+            'a member descriptor (or its '  # (see assert statement above)
+            'equivalent) is not a method descriptor')
+        self.assertFalse(
+            inspect.ismethoddescriptor(getset_descriptor),
+            'a getset descriptor (or its '  # (see assert statement above)
+            'equivalent) is not a method descriptor')
```

Comment:
Put `member_descriptor = Slotermeyer.__dict__['a_slot']` where you use it, not 30 lines above (and you can remove the comment). Ideally:

```py
if hasattr(types, 'MethodDescriptorType'):
	self.assertFalse(inspect.ismemberdescriptor(Slotermeyer.__dict__['a_slot']),
					 'a slot is not a method descriptor'))
```

(same for the getset).